web/cypress/e2e/rest/service-edit-service-with-upstream.cy.js

```diff
@@ -94,7 +93,6 @@ context('Edit Service with Upstream', () => {
         force: true,
       })
       .should('value', data.ip1);
-    cy.get(selector.input).should('be.disabled');
```

Comment:
This is a relatively old test case. In any case, there will be no input box abandoned. It is meaningless to judge whether it is abandoned